tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -95,6 +95,19 @@ enum class TrtTestMode {
   kDynamicShape = 2
 };
 
+string _DebugString(const TrtTestMode mode){
```

Comment:
This is C++ not python. So we want to delcare the routine as "static", and remove "_" from the name "_DebugString".

Reply:
I called it `_DebugString` because otherwise it conflicts with `DebugString` defined in `utils.cc`

Reply:
Right I did this to avoid moving the function outside `namespace convert`. Otherwise the function would mask those defined in `utils.cc`. I'll do that ;)

Reply:
In utils.cc, there isn't a "DebugString(const TrtTestMode mode)", why would renaming this routine to DebugString mask off other DebugString in utils.cc? I was suggesting "static" to make it a file scope routine, another way is to add an anonymous namespace surrounding it.

Reply:
Would you please address this comment?